solution.txt

```diff
@@ -142,23 +138,24 @@ Exercise 12:
 */
 
 // Complete Exercise 12 below...
-
-let fizz = [];
-let buzz = [];
-let fizzbuzz = [];
-
-nums.forEach(fuzzbizz);
-
-function fuzzbizz(num){
-  if (num % 3 === 0 && num % 5 === 0) {
-    fizzbuzz.push(num);
-  } else if (num % 3 === 0) {
-    fizz.push(num);
-  } else if (num % 5 === 0) {
-    buzz.push(num);
+var fizz = []
+var buzz = []
+var fizzbuzz = []
+
+for (let i=0; i<nums.length ; i++)
+{
+  if ((nums[i]/3)%2===0 && ((nums[i]/5)%2===0)) {
+    fizzbuzz.push(nums[i])
+  }
+  else if ((nums[i]/3)%2===0) {
+    fizz.push(nums[i])
+  }
+  else if ((nums[i]/5)%2===0) {
+    buzz.push(nums[i])
   }
 }
```

Comment:
nice work figuring out that `if / else if` needs to check against 3 and 5 *first* in order to work properly! that part trips people up sometimes.